docs/pages/server-access/getting-started.mdx

```diff
@@ -204,35 +230,57 @@ cluster through Teleport SSH.
 
 ### Display cluster resources
 
-`tele-admin` can now execute the following to find the cluster's Node names,
+`tele-admin` can now execute the following to find the cluster's SSH Service names,
 which are used for establishing SSH connections:
 
    ```code
    # Display cluster resources
    $ tsh ls
    ```
 
+<ScopedBlock scope={["oss", "enterprise"]}>
+
    In this example, the bastion host Node is located on the bottom line below:
 
    ```txt
    Node Name        Address        Labels
    ---------------- -------------- --------------------------------------
-   ip-172-31-35-170 ⟵ Tunnel
+   ip-172-31-35-170 ⟵ Tunnel       env=example, hostname=ip-172-31-35-170
    ip-172-31-41-144 127.0.0.1:3022 env=example, hostname=ip-172-31-41-144
    ```
+</ScopedBlock>
+<ScopedBlock scope={["cloud"]}>
 
-### Connect to a Node
+   ```txt
+   Node Name        Address        Labels
+   ---------------- -------------- --------------------------------------
+   ip-172-31-35-170 ⟵ Tunnel       env=example, hostname=ip-172-31-41-144
+   ```
+</ScopedBlock>
+
+### Connect to a SSH Service
 
+<ScopedBlock scope={["oss", "enterprise"]}>
 `tele-admin` can SSH into the bastion host Node by running the following command locally:
 
 ```code
 # Use tsh to ssh into a Node
 $ tsh ssh root@ip-172-31-41-144
 ```
+</ScopedBlock>
+<ScopedBlock scope={["cloud"]}>
+`tele-admin` can SSH into the Node by running the following command locally:
```

Comment:
In the other scope we referred to the target as "the bastion host" and here we refer to it as "the Node." Any reason not to be consistent?